tests/driver_bmx280/main.c

```diff
@@ -36,21 +38,23 @@ int main(void)
 
     puts("BMX280 test application\n");
 
-    printf("+------------Initializing------------+\n");
+    puts("+------------Initializing------------+");
     result = bmx280_init(&dev, &bmx280_params[0]);
-    if (result == -1) {
-        puts("[Error] The given i2c is not enabled");
-        return 1;
+    switch (result) {
```

Comment:
Nit:

```suggestion
    switch (bmx280_init(&dev, &bmx280_params[0])) {
```

and the result variable can be removed.